### PR TITLE
Massive update for Italian language

### PR DESCRIPTION
**Pull request by davi92**
- Massive update for Italian language for all application
- Fix language code for support Italian Switzerland(it-CH), Italian San marino(it-SM) and Italian Vatican (it-VA)

### CODE REVIEW

**Comment by binarymaster** [comment]
Cc @reactos/lang-italian something massive here, need review.

**Review by binarymaster — CHANGES_REQUESTED** [review]
Please fix line-endings changes to make the diff look normal.

File: BOINC.txt [hunk 1]
```diff
@@ -19,3 +19,4 @@ Description = BOINC es una plataforma de software libre destinado a la computaci
 
 [Section.10]
 Description = BOINC è una piattaforma software open source per l'elaborazione che utilizza risorse volontarie.
```
Comment (ElCresh) [inline BOINC.txt:21]: I think is bettere to emphatize with "elaborazione distribuita"

File: ac97forvirtualbox.txt [hunk 2]
```diff
@@ -1,88 +1,88 @@
...
+
+[Section.0a]
+Name = Controlador AC'97 para VirtualBox
+License = Desconocida
+Description = Controlador de audio para dispositivos compatibles con AC'97. Se debe descomprimir en la carpeta «ReactOS» y reiniciar dos veces.
+
+[Section.10]
+Name = Driver AC97 per VirtualBox
+License = Sconosciuta
+Description = Driver audio per dispositivi compatibili con AC'97. Decomprimere nella cartella ReactOS e riavviare due volte.
```
Comment (ElCresh) [inline ac97forvirtualbox.txt:32]: I think should be maintained as "ReactOS" (with quotes) ora << ReactOS>> like in spanish translation.

**Comment by GeoB99** [comment]
@davi92 Do you plan to continue work on this, since the PR is still draft? I plan to take it for a review someday.

**Comment by learn-more** [comment]
I'd say in it's current state this is not reviewable.

**Comment by davi92** [comment]
> @davi92 Do you plan to continue work on this, since the PR is still draft? I plan to take it for a review someday.

    I tried to convert the files to the required format, but as soon as I load them github treats them as binary files.
Sorry for bad english.

**Comment by learn-more** [comment]
Loads of bogus files, and no update from author.

**Pull request closed without merging**